Mathlib/SetTheory/Cardinal/Cofinality.lean

```diff
@@ -303,23 +303,23 @@ theorem lsub_lt_ord {ι} {f : ι → Ordinal} {c : Ordinal} (hι : #ι < c.cof)
     (∀ i, f i < c) → lsub.{u, u} f < c :=
   lsub_lt_ord_lift (by rwa [(#ι).lift_id])
 
-theorem cof_sup_le_lift {ι} {f : ι → Ordinal} (H : ∀ i, f i < sup.{u, v} f) :
-    cof (sup.{u, v} f) ≤ Cardinal.lift.{v, u} #ι := by
+theorem cof_sup_le_lift {ι} {f : ι → Ordinal} (H : ∀ i, f i < iSup f) :
```

Comment:
```suggestion
theorem cof_iSup_le_lift {ι : Sort*} {f : ι → Ordinal} (H : ∀ i, f i < iSup f) :
```
or please specify universes

Reply:
A lot of theorems in that file can now be generalized to other universes. Except, this can't be done fully just yet as a lot of them are written in terms of `lsub`, which we still need to refactor out (or generalize, as I think this file in particular showcases that it is a useful concept). I'd rather not tweak any of this until then.